CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * [CHANGE] Fix test warning related to `cucumber_opts` declaration (by [@faisal][])
 * [BUGFIX] Stop using long-deprecated MiniTest module name, removed in 5.19.0 (by [@faisal][])
+* [CHANGE] Add rexml dependency for Ruby 3.0.0+ (by [@fbuys][])
```

Comment:
Did you add yourself at the bottom of the file?

Ref:
https://github.com/whitesmith/rubycritic/pull/467/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR435

Reply:
Checks failing because https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecrubyversionglobalsusage -> https://rubystyle.guide/#no-ruby-version-in-the-gemspec

Is checking the Ruby version worth it? I believe Bundler should use the installation-provided rexml if we don't care about its version. If we do care about the rexml version then we should care about it regardless of Ruby version.